packages/vanilla/src/complex/array/index.ts

```diff
@@ -11,7 +11,9 @@ export {
   ArrayControlRenderer
 };
 
-export default registerStartupRenderer(
+registerStartupRenderer(
   rankWith(2, isArrayObjectControl),
   ArrayControlRenderer
```

Comment:
why not connected?

Reply:
Same as with the material array renderer, default export is already connected.